Super starter v2

letsleroyu opened the pull request:
Updated superStarter.sol 

- Removed all references to Upgradable, Initializaible
- Changed safeTranfserFrom to safeTransfer in the necessary places.
- Added/changed the admin functions
- Added ReentrancyGuard

TimTinkers commented on contracts/SuperStarter.sol:
```diff
@@ -191,33 +195,33 @@ contract SuperStarter is Initializable, ContextUpgradeable, OwnableUpgradeable {
       @param _token The token to sweep the balance from.
     */
-    function sweep(IERC20 _token) external onlyOwner {
+    function sweep(IERC20 _token, address _recepient) external onlyOwner {
```
Instead of reimplementing this here, let's inherit this from `Sweepable`.

letsleroyu commented:
Just updated SuperStater.sol contract. 
I also changed the error text in the sweep function, it seems to me because the Sweep contract is used not only in MintShop, it is better to make the text more informative.

TimTinkers commented:
> 
> 
> Just updated SuperStater.sol contract.
> I also changed the error text in the sweep function, it seems to me because the Sweep contract is used not only in MintShop, it is better to make the text more informative.

Excellent, and good catch.
